modules/ROOT/pages/dataweave-cookbook-pass-xml-attributes.adoc

```diff
@@ -0,0 +1,56 @@
+= Pass XML Attributes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping
+
+You might want to pass attributes from within an XML tag input source to the target output XML. The DataWeave example shows how to pass the XML attributes in the `NameDetail` field of the input source payload to the `TargetName` field of the XML output.
+//LINK TO DW 1.0 EXAMPLES:
+include::partial$dataweave1-links.adoc[tag=dataweave1Examples]
+
+
+This example uses:
+- `.*` multivalue selector that returns an array of NameDetail tags as its elements.
```

Comment:
"NameDetail tags as its elements" -> "`NameDetail`  elements".  tags are the same things as elements